pallets/attestation/Cargo.toml

```
@@ -3,8 +3,8 @@ authors = ["KILT <info@kilt.io>"]
 description = "Enables adding and revoking attestations."
 edition = "2021"
 name = "attestation"
-repository = "https://github.com/KILTprotocol/kilt-node"
-version = "1.7.1"
+repository = "https://github.com/KILTprotocol/mashnet-node"
```

I suppose this is an inherent error from my update. There are many more. But we can handle this in a separate PR.

```suggestion
repository = "https://github.com/KILTprotocol/kilt-node"
```